package.json

```diff
@@ -26,13 +26,18 @@
   },
   "homepage": "https://github.com/fastify/fastify-redis#readme",
   "devDependencies": {
-    "fastify": "^2.0.0",
-    "standard": "^12.0.0",
-    "tap": "^12.0.1"
+    "fastify": "^2.3.0",
+    "standard": "^12.0.1",
+    "tap": "^12.6.6"
   },
   "dependencies": {
-    "fastify-plugin": "^1.2.0",
-    "ioredis": "^4.0.0",
+    "fastify-plugin": "^1.5.0",
+    "ioredis": "^4.9.0",
     "redis": "^2.8.0"
```

Comment:
I think this redis dependency is spurios. Can you remove it as well?

Reply:
Good catch. ^^
I moved it to the dev-dependencies as it is used to create a client and then test that it won't be automatically shut down by the `onClose` hook.